documentation/automating_windows_targets.md

```diff
@@ -0,0 +1,433 @@
+    file — named after the script, but with a `.json` extension, in this
+    example, `restart_service.json`.
+1.  Add the following content to the new task metadata file:
+
+    ```json
+    {
+      "puppet_task_version": 1,
+      "supports_noop": false,
+      "description": "Stop or restart a service or list of services on a target.",
+      "parameters": {
+        "service": {
+          "description": "The name of the service, or a list of service names to stop.",
+          "type": "Variant[Array[String],String]"
+        },
+        "norestart": {
+          "description": "Immediately restart the services after start.",
+          "type": "Optional[Boolean]"
+        }
+      }
+    }
+    ```
+
+1.  Save the task metadata file and navigate back to the `bolt-guide` directory.
+
+    You now have two files in the `gsg` module’s `tasks` directory:
+    `restart_service.ps1` and `restart_service.json` -- the script is officially
+    converted to a Bolt task. Now that it’s converted, you no longer need to
+    specify the file extension when you call it from a Bolt command.
+1.  Validate that Bolt recognizes the script as a task:
+    ```
+    bolt task show gsg::restart_service
+    ```
+
+    ![](bolt_PS_2.png)
+
+    Congratulations! You’ve successfully converted the `restart_service.ps1`
+    script to a Bolt task.
+
+1.  Execute your new task:
+    ```
+    bolt task run gsg::restart_service service=W32Time --targets windows
+    ```
+
+    ![](bolt_PS_3.png)
+
+    **Note:** `--targets windows` refers to the name of the group of targets
+    that you specified in your inventory file. For more information, see
+    [Specify targets](running_bolt_commands.md#adding-options-to-bolt-commands).
+
+
+## Deploy a package with Bolt and Chocolatey
+
+You can use Bolt with Chocolatey to deploy a package on a Windows node by
+writing a [Bolt plan](writing_plans.md) that installs Chocolatey and uses
+Puppet's Chocolatey package provider to install a package. This is all done
+using content from the [Puppet Forge](https://forge.puppet.com).
+
+> **Before you begin**
+>
+>- Install [Bolt](https://puppet.com/docs/bolt/latest/bolt_installing.html) and
+>  the [Puppet Development Kit
+>  (PDK)](https://www.puppet.com/docs/pdk/1.x/pdk_install.html).
+>- Ensure you have Powershell and Windows Remote Management (WinRM) access.
+
+In this example, you:
+
+- Build a project-specific configuration using a Bolt project directory and PDK.
+- Download module content from the Puppet Forge.
+- Write a Bolt plan to apply Puppet code and orchestrate the deployment of a
+  package resource using the Chocolatey provider.
+
+### 1. Build a project-specific configuration
+
+Bolt runs in the context of a [Bolt project
+directory](bolt_project_directories.md). This directory contains all of the
+configuration, code, and data loaded by Bolt.
+
+1. Create a module called `bolt_choco_example`
+
+```
```

Comment:
From this point of the doc onwards, the ordered lists go all funky. I think the content between each list item needs to be indented. I can fix this in a separate ticket if you like. This doc needs updates anyway.